src/coreclr/src/pal/src/configure.cmake

```diff
@@ -8,7 +8,7 @@ include(CheckTypeSize)
 include(CheckLibraryExists)
 
 if(CLR_CMAKE_TARGET_FREEBSD)
-  set(CMAKE_REQUIRED_INCLUDES /usr/local/include)
+  set(CMAKE_REQUIRED_INCLUDES ${CROSS_ROOTFS}/usr/local/include)
```

Comment:
Dtto, I believe the CROSS_ROOTFS should be added automatically during cross compilation.

Reply:
think ROOTFS_DIR/usr/include is but ROOTFS_DIR/usr/local/include is not. This is because some dependencies are coming from extra package.